seed/seed.js

```diff
@@ -5,63 +5,169 @@
 const childProcess = require('child_process');
 const path = require('path');
 
+const { getAllCourses } = require('../resources/grabs');
 const models = require('../server/db/index.js');
 const { db } = models;
 const userList = require('./seedlings/userSeed');
-const groupList = require('./seedlings/groupSeed');
+const { groupList } = require('./seedlings/groupSeed');
 const eventList = require('./seedlings/eventSeed');
 const postList = require('./seedlings/postSeed');
+const zipList = require('./seedlings/zipSeed');
+
+const findModel = async (name) => {
+  const topic = await models.Topic.findOne({
+    where: {
+      name
+    }
+  })
+  return topic;
+}
+
+const hasKeys = (obj) => {
```

Comment:
This can definitely be improved. Checkout Object.keys()

Reply:
Yeah, I'm just gonna get rid of this function completely I think. It doesn't serve that much of a purpose.